src/sentry/insights/migrations/0002_backfill_team_starred.py

```diff
@@ -0,0 +1,97 @@
+# Generated by Django 5.2.1 on 2025-06-10 16:25
+import logging
+from typing import cast
+
+from django.db import IntegrityError, migrations, router, transaction
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+from sentry.discover.models import TeamKeyTransaction
+from sentry.insights.models import InsightsStarredSegment
+from sentry.models.organization import Organization
+from sentry.models.organizationmemberteam import (
+    OrganizationMemberTeam as OrganizationMemberTeamType,
+)
+from sentry.models.projectteam import ProjectTeam
```

Comment:
Don't import external models into the migrations. These type hints are wrong, because the types in here are not real models, they're auto generated models that the migration system builds

Reply:
Thanks! I thought it would be easier because without it it there seems to be no type hinting, but if it's not right anyways we shouldn't do it. 👍